security/tls-authorization/README.adoc

```diff
@@ -0,0 +1,268 @@
+= Remote caches with TLS (SSL) encryption and security authorization
+
+**Authors:** Wolf Fink +
+**Technologies:** Infinispan, Hot Rod, Java, SSL/TLS, RBAC +
+**Summary:** Use SSL/TLS certificates to verify Hot Rod client identities and encrypt network traffic to the server. This tutorial also demonstrates how to perform client certificate authentication and use security authorization with client certificates.
+
+== Building instructions and preparation
+
+. Make sure you are using Java 11 at a minimum.
++
+Infinispan Server requires at least Java 11.
+Additionally the example keystores and trust stores in this tutorial use `PKCS12`, which is the recommended standard and the default from Java 11.
+Java8 keytool will prompt for a password which can be prevented by adding `-keypass` to some of the commands. The storetype is already set.
+. Run the command to build the project
+* `$ mvn clean package`
+. Build the example certificates and keystores with the following scripts or directly with the Java keytool commands:
+* `create_unsigned_certificates.sh`
+* `create_signed_certificates.sh`
++
+These scripts create keystores and trust stores in the project directory.
+. Download at least one Infinispan Server instance.
+
+== Unsigned Server certificates
+
+. Copy `server-keystore.pfx` to `$ISPN_HOME/server/conf`.
+. Copy `server/infinispan-uCert.xml` to `$ISPN_HOME/server/conf`.
+. Use the `infinispan-uCert.xml` configuration, changes are
```

Comment:
```suggestion
. Copy `server/infinispan-uCert.xml` to `$ISPN_HOME/server/conf`.
This configuration, `infinispan-uCert.xml`, contains the following changes:
```

Reply:
Probably better not to make "Use the ... configuration` as a step. This is covered in the start the server step.